FBSnapshotTestCaseDemo/FBSnapshotTestCaseDemo/Launch Screen.storyboard

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<document type="com.apple.InterfaceBuilder3.CocoaTouch.Storyboard.XIB" version="3.0" toolsVersion="9531" systemVersion="15C50" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" launchScreen="YES" useTraitCollections="YES" initialViewController="01J-lp-oVM">
```

Comment:
Yes, the launch screen fixes the issue where the app doesn't fill the
screen.

On Fri, Jan 8, 2016 at 12:50 Patrick Chamelo <notifications@github.com>
wrote:

> In FBSnapshotTestCaseDemo/FBSnapshotTestCaseDemo/Launch Screen.storyboard
> <https://github.com/facebook/ios-snapshot-test-case/pull/137#discussion_r49236579>
> :
>
> > @@ -0,0 +1,50 @@
> > +<?xml version="1.0" encoding="UTF-8" standalone="no"?>
> > +<document type="com.apple.InterfaceBuilder3.CocoaTouch.Storyboard.XIB" version="3.0" toolsVersion="9531" systemVersion="15C50" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" launchScreen="YES" useTraitCollections="YES" initialViewController="01J-lp-oVM">
>
> Do we need this?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/facebook/ios-snapshot-test-case/pull/137/files#r49236579>
> .
>